### DISCUSSION TITLE
Kein Lizenzkauf möglich

### DISCUSSION THREAD

**Discussion by dengler5000** [brutella/hkknx-public#170] [General]
Hallo, beim Versuch einer Lizenzbestellung bekomme ich leider immer folgende Fehlermeldung:

Get "https://api.stripe.com/v1/products/prod_JdJpV1PZOv2dfx": x509: certificate has expired or is not yet valid: current time 2021-05-23T13:20:56+02:00 is before 2022-01-11T00:00:00Z.

Wie lässt sich dieser Fehler beheben?

Gruß dengler5000

**Comment by brutella** [comment]
Siehe https://hochgatterer.me/de/hkknx/docs/help/buy-license-error/

**Reply by dengler5000** [reply]
Hat funktioniert, danke :) 
Wenn ich jetzt noch an nen Aktionscode kommen könnte .... :D
